payments-core/src/main/java/com/stripe/android/payments/core/authentication/threeds2/Stripe3ds2TransactionViewModel.kt

```diff
@@ -288,52 +310,30 @@ internal sealed class NextStep {
 }
 
 internal class Stripe3ds2TransactionViewModelFactory(
```

Comment:
Did you look into using Hilt? Why can't we use it? https://dagger.dev/hilt/view-model.html

Reply:
Hilt [requires](https://dagger.dev/hilt/application) access to `@Application`, which we don't have access to